zebra-chain/src/block.rs

```diff
@@ -1,9 +1,19 @@
 //! Definitions of block datastructures.
 
-use chrono::{DateTime, Utc};
+#[cfg(test)]
+pub mod test_vectors;
+#[cfg(test)]
+mod tests;
```

Comment:
This can be in a later PR, but if `block` has submodules, do we want to try to arrange the submodules of `block` to match `transaction`, splitting out into submodules, e.g., `header`, `serialize`, `tests`, etc, and then re-exporting components from `block`?

Reply:
I think so yes 👍

Reply:
Tracked in #220